src/languages/singlestoredb/singlestoredb.formatter.ts

```diff
@@ -221,7 +224,7 @@ const reservedJoins = expandPhrases([
   'JOIN',
   '{LEFT | RIGHT | FULL} [OUTER] JOIN',
   '{INNER | CROSS} JOIN',
-  'NATURAL {LEFT | RIGHT} [OUTER] JOIN',
+  '{LEFT | RIGHT} [OUTER] JOIN',
```

Comment:
Why did you opt to remove NATURAL JOIN support?

- [the official docs](https://docs.singlestore.com/managed-service/en/reference/sql-reference/data-manipulation-language-dml/select.html#join-and-subqueries) say it's supported.
- I even tested it out and it worked just fine.

Reply:
Perhaps you made the mistake of testing `NATURAL JOIN`, which isn't supported, while e.g. `NATURAL LEFT JOIN` is.